projects/step-app/src/styles.scss

```diff
@@ -1,5 +1,198 @@
 @use 'projects/step-frontend/styles/common-mixins';
 @import 'projects/step-frontend/styles/fonts-setup';
 
+/* Design to align with new step layout */
```

Comment:
I think it would be nicer to put these rules to as a part of the theme mixin.
So it could be applied in both versions and use the same configuration based on scss variables.